pkg/apic/provisioning/definitions.go

```diff
@@ -29,16 +29,30 @@ const (
 	RequestTypeRenew
 )
 
-// status - the status of the request
-type status int
+func (c RequestType) String() string {
+	return map[RequestType]string{
+		RequestTypeProvision: "provision",
+		RequestTypeRenew:     "renew",
+	}[c]
+}
+
+// Status - the Status of the request
+type Status int
 
 const (
 	// Success - request was successful
-	Success status = iota + 1
+	Success Status = iota + 1
 	// Failed - request failed
 	Failed
 )
 
+func (c Status) String() string {
+	return map[Status]string{
+		Success: "success",
+		Failed:  "failed",
```

Comment:
These constants are getting used for building the status subresource, so the value should correspond to "Success" and "Error"(instead of "failed" for error case)